swagger.json

```diff
@@ -1821,6 +1821,69 @@
             }
           }
         },
+        "/fleet/vehicles/stats": {
+            "parameters": [
+                {
+                  "$ref": "#/parameters/accessTokenParam"
+                },
+                {
+                  "name": "startMs",
+                  "description": "Time in Unix epoch milliseconds for the start of the query (cannot exceed 1 hour)",
+                  "required": true,
+                  "in": "query",
+                  "type": "integer",
+                  "format": "int64"
+                },
+                {
+                  "name": "endMs",
+                  "description": "Time in Unix epoch milliseconds for the end of the query (cannot exceed 1 hour)",
```

Comment:
Same as above. Would we always return the most recent timeseries value if we passed graphql (nowMs, nowMs) as the (startMs, endMs) values? If so, that would be an ideal default.